charts/cluster-autoscaler/Chart.yaml

```diff
@@ -11,4 +11,4 @@ name: cluster-autoscaler
 sources:
   - https://github.com/kubernetes/autoscaler/tree/master/cluster-autoscaler
 type: application
-version: 9.24.0
+version: 9.24.1
```

Comment:
We're bumping the app version an entire minor version, can we please make the chart version bump also a minor version bump.

Reply:
I think that the minor version of the chart and the minor version of the image were supposed to match: `9.24.x` = `1.24.x`. Could it be that the numbers are just coincidentally close now?

Reply:
Fix: https://github.com/kubernetes/autoscaler/pull/5517/commits/9e09f529fee9bc67c7f47c0aec6ae1b4221f0c9c

Reply:
Yep, just a coincidence at this point!